### PR TITLE
feat(benchmark): Add filter flag

### PR DESCRIPTION
**Pull request by tomi**
## Summary

- feat(benchmark): Enable filtering scenarios with -f flag
- docs(benchmark): Update readme

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/benchmark/src/scenario/scenario-loader.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ export class ScenarioLoader {
 		const scenarios: Scenario[] = [];
 
 		for (const folder of scenarioFolders) {
+			if (filter && !folder.toLowerCase().includes(filter.toLowerCase())) {
+				continue;
+			}
```
Comment (valya) [inline packages/@n8n/benchmark/src/scenario/scenario-loader.ts:23]: Could we maybe make this a regex at some point?

**Review by valya — APPROVED** [review]

File: packages/@n8n/benchmark/src/scenario/scenario-loader.ts (hunk 1 above)
Comment (tomi) [reply packages/@n8n/benchmark/src/scenario/scenario-loader.ts:23]: I thought about allowing glob pattern for it, but at least for now the use case would be to filter only a single scenario, I didn't want to complicate it. Let's revisit this when there's a need for this

Comment (valya) [reply packages/@n8n/benchmark/src/scenario/scenario-loader.ts:23]: Yeah, that's fine. I just know I use the partial matches in Jest all the time, so if I was running benchmarks, I'd probably want that as well.

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**